test/commandLine/commandLineTest.js

```diff
@@ -0,0 +1,662 @@
+/* eslint-env mocha */
+
+const assert = require('assert')
+const fs = require('fs')
+const fse = require('fs-extra')
+const path = require('path')
+const rimraf = require('rimraf')
+const fork = require('child_process').fork
+
+describe('Command Line Tests', function () {
+  const appDir = path.join(__dirname, '../app/commandLineTest')
+
+  before(function () {
+    fse.ensureDirSync(path.join(appDir))
+    let dataIn = `const path = require('path')
+    const appDir = path.join(__dirname, '../app/commandLineTest')
+    const app = require('../../../roosevelt')({
+      appDir: appDir,
+      generateFolderStructure: false,
+    })
+    if (process.send) {
+      process.send(app.expressApp.get('params'))
+    }
+    `
+
+    fs.writeFileSync(path.join(appDir, 'app.js'), dataIn)
+  })
+
+  after(function (done) {
+    rimraf(appDir, (err) => {
+      if (err) {
+        throw err
+      } else {
+        done()
+      }
+    })
+  })
+
+  /* check individual flags */
```

Comment:
Add a category for soon to be deprecated flags and move them all up there together. That way when we deprecate them we can dump their tests too without having to dig. Also avoid using soon to be deprecated flags in the combination tests for this same reason.